remix_exercises/Challenge0.sol

```diff
@@ -0,0 +1,14 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface Challenge0 {
```

Comment:
whats the point of this interface? wouldn't you just deploy the `Challenge0` contract and hit the callMe button?

Reply:
I was thinking we'd deploy the contracts and have them call them using the interface, that way (as you mention below) they can read the public secret value but would have to access storage to find the private value.